src/utils.js

```diff
@@ -16,13 +19,23 @@ export function randomString(
   return result;
 }
 
+/**
+ * [safeSlot description]
+ *
+ * @param  {[type]} h    hyperscript markup from Vue render function
+ * @param  {[type]} slot component slot
+ * @return {[type]}      Slot or div wrapping the slot
+ */
 export function safeSlot(h, slot) {
   return slot && slot.length > 1 ? h('div', slot) : slot;
 }
 
 /**
- * @param {HTMLElement} el
- * @param {Event} event
+ * [applyFocusTrap description]
```

Comment:
Replace `[applyFocusTrap description]` with a useful description of the function.